fedimintd/umbrel-app.yml

```
@@ -0,0 +1,25 @@
+manifestVersion: 1
+id: fedimintd
```

It may make sense to just call it `fedimint`
```suggestion
id: fedimint
```

---

My original reasoning behind using `fedimintd` is that in the future we would also like to add additional Fedimint services to Umbrel, like the Lightning Gateway. I felt this was more clear and we did the same thing on the Start9 package too.

If you feel strongly though, I can change it.

---

Going to keep this as is unless there are strong feelings to change it.